README.md

```diff
@@ -1,2 +1,169 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li>Ruby</li>
+    <li>Rails</li>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Test-Driven Development
+- Class implementation with methods
+- Data Preserved in .json files
+- Unit tests created with Rspec
+- Backend PGSQL tables
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+git clone https://github.com/Elhambasir/budget-app.git
+
+```
+
+### Install
+
+To run the project, execute the folllowing in a terminal.
+
+```sh
+cd bugdet-app
+gem install
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Example command:
+
+```sh
+  rails server
+```
+
+
+### Run tests
+To run tests, run the following command:
+npm test
+Example command:
+```sh
+  npx stylelint "**/*.{css,scss}"
+  rubocop .
+  rspec spec
+```
+### Deployment <a name="deployment"></a>
+
+- [Live](comming soon)
+
+- Demo [video](https://www.loom.com/share/a0dd5f383ad241f3a3e7681e60fb4350?sid=eb746e2f-4ce6-468e-9a54-dcc78763c837)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **BasirElham**
+
+- GitHub: [BasirElham](https://github.com/Elhambasir)
+- LinkedIn: [BasirElham](https://www.linkedin.com/in/basirelhamahmadi)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Save to file
+- Styling data adding pages
+- Joined tables
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Elhambasir/budget-app/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, leave a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse community and The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) for such a great design
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/Elhambasir/budget-app/assets/77069011/a9d7376a-463e-4561-b599-d55771286c92)

Please use this section to give the appropriate credits to the design author! 🤓 🔧

Reply:
I am sorry but I did not get what you mean. What is wrong with my acknowledgments? Please give me more details or clue so that can help me understand.